resources/integration_test/tests/sqltables.yml

```diff
@@ -1,5 +1,38 @@
 tests:
-  - description: SQLTablesW all catalogs with TableType
+  - description: SQLTablesW SQL_ALL_CATALOGS lists valid catalogs
+    test_definition: [ "sqltablesw", "%", 1, "", 0, "", 0, "", 0 ]
+    db: integration_test
+    expected_result:
+      - [ "db2", null, null, null, "" ]
+      - [ "integration_test", null, null, null, "" ]
+      - [ "integration_test_2", null, null, null, "" ]
+      - [ "tdvt", null, null, null, "" ]
+    expected_bson_type: [ "string", "string", "string", "string", "string" ]
+    expected_case_sensitive: [ "", "", "", "", "" ]
+    expected_catalog_name: [ "", "", "", "", "" ]
+    expected_column_name: [ "TABLE_CAT", "TABLE_SCHEM", "TABLE_NAME", "TABLE_TYPE", "REMARKS" ]
+    expected_display_size: [ 0, 0, 0, 0, 0 ]
+    expected_length: [ 0, 0, 0, 0, 0 ]
+    expected_is_searchable: [ 3, 3, 3, 3, 3 ]
+    expected_is_unsigned: [ 1, 1, 1, 1, 1 ]
+    expected_sql_type: [ -9, -9, -9, -9, -9 ]
+    expected_precision: [ 0, 0, 0, 0, 0 ]
+    expected_scale: [ 0, 0, 0, 0, 0 ]
+    expected_nullability: [ 0, 1, 1, 1, 1 ]
+
+  - description: SQLTablesW SQL_ALL_SCHEMA returns empty list
+    test_definition: [ "sqltablesw", "", 0, "%", 1, "", 0, "", 0 ]
+    db: integration_test
+    expected_result: []
+
+  - description: SQLTablesW SQL_ALL_TABLE_TYPES lists valid types
+    test_definition: [ "sqltablesw", "", 0, "", 0, "", 0, "%", 1 ]
+    db: integration_test
+    expected_result:
+      - [ null, null, null, "TABLE", null ]
```

Comment:
Looks like we have a small bug here, the 5th value should be an empty string `""`.  
Would you please create a ticket for that and add a skip reason here with the JIRA value?

Reply:
Is there any preference between adding a ticket and making the chance in this pr? I think it will be a one line difference either way, so happy to do whichever makes more sense.

Reply:
And thanks for catching this discrepancy 😄 Looking at the spec again, I think this test might be correct and it is the `SQL_ALL_CATALOGS` that needs updating: 
```
If TableType is SQL_ALL_TABLE_TYPES and CatalogName, SchemaName, and TableName are empty strings, the result set contains a list of valid table types for the data source. (All columns except the TABLE_TYPE column contain NULLs.)
```
Either way, the above question still stands on if we should update as part of this PR.

Reply:
Good call on double-checking the spec.  
Let's add the change in this PR if it is not too much effort (sound like it shouldn't be).  I ran a quick test of setting remarks to null and table listing worked as expected through Power BI.

Reply:
Sounds good, I will add this